pkg/quarkus/v1alpha/api.go

```diff
@@ -116,3 +158,82 @@ func (p *createAPISubcommand) InjectResource(res *resource.Resource) error {
 
 	return nil
 }
+
+// findOldFilesForReplacement verifies marker (## marker) and if it found then merge new api CRD file to the odler logic
+func findOldFilesForReplacement(path, newfile string) bool {
+
+	f, err := os.Open(path)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// remember to close the file at the end of the program
+	defer f.Close()
+
+	// read the file line by line using scanner
+	scanner := bufio.NewScanner(f)
+	var foundMarker bool
+	for scanner.Scan() {
+		// do something with a line
+		if scanner.Text() == "## marker" {
+			foundMarker = true
+			break
+		}
+	}
+
+	if foundMarker {
+		scanner.Scan()
+		catLine := scanner.Text()
+
+		splitByPipe := strings.Split(catLine, "|")
+
+		finalString := strings.TrimSuffix(strings.TrimPrefix(strings.TrimSpace(splitByPipe[0]), "cat"), "target/kubernetes/kubernetes.yml")
+
+		updatedLine := "	" + "cat" + finalString + newfile + " target/kubernetes/kubernetes.yml" + " |" + splitByPipe[1]
+		// fmt.Printf("merge : %s\n", merge)
+
+		if err := scanner.Err(); err != nil {
+			log.Fatal(err, "Failed to scan the line from Makefile")
+			return false
+		}
+
+		// ReplaceInFile replaces all instances of old with new in the file at path.
+		err = util.ReplaceInFile(path, catLine, updatedLine)
+		if err != nil {
+			log.Fatal(err, "Failed to Replace in Makefile")
+			return false
+		}
+	}
+
+	return foundMarker
+}
+
+const (
+	makefileBundleCRDFile = `target/kubernetes/%[1]s.%[2]s-%[3]s.yml`
+)
+
+const (
+	makefileBundleVarFragment = `
+##@Bundle
+.PHONY: bundle
+bundle:  ## Generate bundle manifests and metadata, then validate generated files.
+## marker
+	cat target/kubernetes/%[1]s.%[2]s-%[3]s.yml target/kubernetes/kubernetes.yml | operator-sdk generate bundle -q --overwrite --version 0.1.1 --default-channel=stable --channels=stable --package=%[4]s
+	operator-sdk bundle validate ./bundle
+	
+.PHONY: bundle-build
+bundle-build: ## Build the bundle image.
+	docker build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
+	
+.PHONY bundle-push
+bundle-push: ## Push the bundle image.
+	docker push $(BUNDLE_IMG)
+`
+)
+
+const (
+	makefileBundleImageFragement = `
+IMAGE_TAG_BASE ?= %[1]s/%[2]s
+BUNDLE_IMG ?= $(IMAGE_TAG_BASE)-bundle:v$(VERSION)
```

Comment:
VERSION isn't defined anywhere. And if I don't pass any variables in I get an image: 
`example.com/memcached-quarkus-operator-bundle                                  v                                                     c13e715c04f4   8 minutes ago   6.38kB
`

Reply:
So it's tagged with just a `v`.

Reply:
Yes added version at the top as `VERSION ?= 0.0.1`

Reply:
I will also run the entire process with `run bundle` before tomorrow's demo.